src/routes/(auth)/login/+page.svelte

```diff
@@ -14,26 +14,25 @@
       password: ''
     },
     validationSchema: Yup.object({
-      email: Yup.string().email().required(),
-      password: Yup.string().required()
+      email: Yup.string().email(),
+      password: Yup.string()
     }),
     onSubmit: async ({ email, password }) => {
-      setTimeout(async () => {
-        const basicAuth = createHeader(email, password);
-        const response = await fetch('/login', {
-          method: 'POST',
-          headers: {
-            Authorization: basicAuth
-          }
-        });
-
-        if (response.ok) {
-          notification.notifySuccess('Logged in successfully');
-          window.location.pathname = '/';
-        } else {
-          notification.notifyFailure('Failed to login, try again');
+      const basicAuth = createHeader(email, password);
+      const request = await fetch('/login', {
+        method: 'POST',
+        headers: {
+          Authorization: basicAuth
         }
-      }, 2000);
+      });
+
+      if (request.ok) {
+        notification.notifySuccess('Logged in successfully');
+        window.location.pathname = '/';
+      } else {
+        const response = await request.json();
+        notification.notifyFailure(response.message);
```

Comment:
Please build a user friendly message based on error code. Do not reuse server side error message intended for developer.